test/driver.cpp

```diff
@@ -56,18 +59,71 @@ int main(int argc, char* argv[])
 
 
 	try {
-		runTests();
+		run_suites(options);
 	}
-	catch (const std::string& msg) {
-		logLine(msg.data());
-		if (options.fom == file_open_mode::no_file)
-			show_error(msg.data());
+	catch (const cmd_line_error& cle) {
+		show_error_and_usage(cle.what(), argv[0]);
+		return -5;
+	}
+	catch (const std::exception& e) {
+		show_error((std::string{ "Unexpected error: " } +e.what()).data());
```

Comment:
```suggestion
		show_error((std::string{ "Unexpected error: " } + e.what()).data());
```

Reply:
Thanks for the suggestion, but would please state the issue? I'm unable to see it.

Reply:
It seems that @cmcmone is suggesting a formatting change; adding a space after the concatenation operator for clarity and readability. @smurthys

Reply:
Please see my response [here](https://github.com/sigcpp/stl-lite/pull/18#discussion_r439748522).